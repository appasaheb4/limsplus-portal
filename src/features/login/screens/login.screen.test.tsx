import React from 'react';
import {render} from '@testing-library/react';
import {Login} from '.';

describe('Login Screen', () => {
  it('render login correctly', () => {
    const login = render(<Login />);
    expect(login).toMatchSnapshot();
  });
});
