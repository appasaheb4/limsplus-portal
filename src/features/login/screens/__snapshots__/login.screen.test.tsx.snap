// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Screen render login correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-yellow-300 h-screen container-fluid"
      >
        <div
          class="items-center pt-4 mb-4 row"
        >
          <div
            class="col-md-7"
          >
            <div
              class="flex flex-col justify-center items-center"
            >
              <img
                alt="logo"
                class="w-20 h-15"
                src="applogo.png"
              />
              <div
                class="mt-2"
              >
                <div
                  class="carousel slide"
                >
                  <ol
                    class="carousel-indicators"
                  />
                  <div
                    class="carousel-inner"
                  />
                  <a
                    class="carousel-control-prev"
                    href="#"
                    role="button"
                  >
                    <span
                      aria-hidden="true"
                      class="carousel-control-prev-icon"
                    />
                    <span
                      class="sr-only"
                    >
                      Previous
                    </span>
                  </a>
                  <a
                    class="carousel-control-next"
                    href="#"
                    role="button"
                  >
                    <span
                      aria-hidden="true"
                      class="carousel-control-next-icon"
                    />
                    <span
                      class="sr-only"
                    >
                      Next
                    </span>
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div
            class="col-md-5"
          >
            <div
              class="flex flex-col items-center"
            >
              <img
                alt="logo"
                class="w-20 h-15  self-center"
                src="applogo.png"
              />
              <div
                class="flex flex-col p-3 mt-2 rounded-md bg-black shadow-sm w-full"
              >
                <div
                  class="flex mt-2 justify-center items-center"
                >
                  <label
                    class="font-bold text-3xl text-white"
                  >
                    Login
                  </label>
                </div>
                <div>
                  <span
                    class="flex flex-col space-y-4    flex-1 items-stretch"
                  >
                    <div>
                      <label
                        class="text-gray-700 block text-3xs font-medium  mb-1"
                        for="userId"
                        style="color: white;"
                      >
                        User Id
                      </label>
                      <input
                        autocomplete="given-name"
                        class="undefined leading-4 p-2  focus:outline-none focus:ring  block w-full shadow-sm sm:text-base  border-2  border-gray-300 rounded-md"
                        data-testid="INPT"
                        id="userId"
                        name="userId"
                        placeholder="UserId"
                        type="text"
                        value=""
                      />
                    </div>
                    <div>
                      <label
                        class="text-gray-700 block text-3xs font-medium  mb-1"
                        for=""
                        style="color: white;"
                      >
                        Password
                      </label>
                      <input
                        autocomplete="given-name"
                        class="undefined leading-4 p-2  focus:outline-none focus:ring  block w-full shadow-sm sm:text-base  border-2  border-gray-300 rounded-md"
                        data-testid="INPT"
                        placeholder="Password"
                        type="password"
                        value=""
                      />
                    </div>
                    <div>
                      <label
                        class="text-gray-700 block text-3xs font-medium  mb-1"
                        for=""
                        style="color: white;"
                      >
                        Lab
                      </label>
                      <select
                        class="leading-4 p-2 focus:outline-none focus:ring block w-full shadow-sm sm:text-base border-2 border-gray-300 rounded-md"
                      >
                        <option>
                          Select
                        </option>
                      </select>
                    </div>
                    <div>
                      <label
                        class="text-gray-700 block text-3xs font-medium  mb-1"
                        for=""
                        style="color: white;"
                      >
                        Role
                      </label>
                      <select
                        class="leading-4 p-2 focus:outline-none focus:ring block w-full shadow-sm sm:text-base border-2 border-gray-300 rounded-md"
                      >
                        <option>
                          Select
                        </option>
                      </select>
                    </div>
                  </span>
                  <br />
                  <span
                    class="flex flex-row space-x-3   justify-center  items-center"
                  >
                    <div
                      class="undefined"
                    >
                      <button
                        class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
                        type="button"
                      >
                        <svg
                          class="false h-5 w-5 text-white -ml-1 mr-2"
                          fill="none"
                          stroke="currentColor"
                          viewBox="0 0 24 24"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M7.197,16.963H7.195c-0.204,0-0.399-0.083-0.544-0.227l-6.039-6.082c-0.3-0.302-0.297-0.788,0.003-1.087 C0.919,9.266,1.404,9.269,1.702,9.57l5.495,5.536L18.221,4.083c0.301-0.301,0.787-0.301,1.087,0c0.301,0.3,0.301,0.787,0,1.087 L7.741,16.738C7.596,16.882,7.401,16.963,7.197,16.963z"
                            fill="none"
                          />
                        </svg>
                        Login
                      </button>
                    </div>
                    <div
                      class="undefined"
                    >
                      <button
                        class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
                        type="button"
                      >
                        <svg
                          class="false h-5 w-5 text-white -ml-1 mr-2"
                          fill="none"
                          stroke="currentColor"
                          viewBox="0 0 24 24"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M6 18L18 6M6 6l12 13"
                            stroke-linecap="round"
                            stroke-linejoin="round"
                            stroke-width="2"
                          />
                        </svg>
                        Clear
                      </button>
                    </div>
                  </span>
                  <h4
                    class="text-center mt-2 text-white"
                  >
                     
                    <b>
                      Note
                    </b>
                    : After 3 invalid login attempts, accounts will be locked.
                  </h4>
                  <h4
                    class="text-center text-white"
                  >
                    In that case contact the Support Team.
                  </h4>
                </div>
                <div
                  class="flex p-4 flex-row items-center justify-around"
                >
                  <div
                    class="flex mt-2 justify-center items-center"
                  >
                    <a
                      class="text-white mr-2"
                      href="#"
                    >
                      Forgot Password
                    </a>
                    <a />
                  </div>
                  <div>
                    <a
                      class="text-white"
                      href="privacy-policy"
                    >
                      Privacy and Policy
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="flex flex-row justify-center"
        >
          <a
            href="https://appho.st/d/VZXlvzKV"
          >
            <img
              class="h-15"
              src="app-ico.png"
            />
          </a>
              
          <a
            href="https://apps.apple.com/us/app/memetoons/id1517184743"
          >
            <img
              class="h-15"
              src="app-ico2.png"
            />
          </a>
        </div>
        <div
          class="container"
        />
        <div
          class="container"
        />
        <div
          class="container"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-yellow-300 h-screen container-fluid"
    >
      <div
        class="items-center pt-4 mb-4 row"
      >
        <div
          class="col-md-7"
        >
          <div
            class="flex flex-col justify-center items-center"
          >
            <img
              alt="logo"
              class="w-20 h-15"
              src="applogo.png"
            />
            <div
              class="mt-2"
            >
              <div
                class="carousel slide"
              >
                <ol
                  class="carousel-indicators"
                />
                <div
                  class="carousel-inner"
                />
                <a
                  class="carousel-control-prev"
                  href="#"
                  role="button"
                >
                  <span
                    aria-hidden="true"
                    class="carousel-control-prev-icon"
                  />
                  <span
                    class="sr-only"
                  >
                    Previous
                  </span>
                </a>
                <a
                  class="carousel-control-next"
                  href="#"
                  role="button"
                >
                  <span
                    aria-hidden="true"
                    class="carousel-control-next-icon"
                  />
                  <span
                    class="sr-only"
                  >
                    Next
                  </span>
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="col-md-5"
        >
          <div
            class="flex flex-col items-center"
          >
            <img
              alt="logo"
              class="w-20 h-15  self-center"
              src="applogo.png"
            />
            <div
              class="flex flex-col p-3 mt-2 rounded-md bg-black shadow-sm w-full"
            >
              <div
                class="flex mt-2 justify-center items-center"
              >
                <label
                  class="font-bold text-3xl text-white"
                >
                  Login
                </label>
              </div>
              <div>
                <span
                  class="flex flex-col space-y-4    flex-1 items-stretch"
                >
                  <div>
                    <label
                      class="text-gray-700 block text-3xs font-medium  mb-1"
                      for="userId"
                      style="color: white;"
                    >
                      User Id
                    </label>
                    <input
                      autocomplete="given-name"
                      class="undefined leading-4 p-2  focus:outline-none focus:ring  block w-full shadow-sm sm:text-base  border-2  border-gray-300 rounded-md"
                      data-testid="INPT"
                      id="userId"
                      name="userId"
                      placeholder="UserId"
                      type="text"
                      value=""
                    />
                  </div>
                  <div>
                    <label
                      class="text-gray-700 block text-3xs font-medium  mb-1"
                      for=""
                      style="color: white;"
                    >
                      Password
                    </label>
                    <input
                      autocomplete="given-name"
                      class="undefined leading-4 p-2  focus:outline-none focus:ring  block w-full shadow-sm sm:text-base  border-2  border-gray-300 rounded-md"
                      data-testid="INPT"
                      placeholder="Password"
                      type="password"
                      value=""
                    />
                  </div>
                  <div>
                    <label
                      class="text-gray-700 block text-3xs font-medium  mb-1"
                      for=""
                      style="color: white;"
                    >
                      Lab
                    </label>
                    <select
                      class="leading-4 p-2 focus:outline-none focus:ring block w-full shadow-sm sm:text-base border-2 border-gray-300 rounded-md"
                    >
                      <option>
                        Select
                      </option>
                    </select>
                  </div>
                  <div>
                    <label
                      class="text-gray-700 block text-3xs font-medium  mb-1"
                      for=""
                      style="color: white;"
                    >
                      Role
                    </label>
                    <select
                      class="leading-4 p-2 focus:outline-none focus:ring block w-full shadow-sm sm:text-base border-2 border-gray-300 rounded-md"
                    >
                      <option>
                        Select
                      </option>
                    </select>
                  </div>
                </span>
                <br />
                <span
                  class="flex flex-row space-x-3   justify-center  items-center"
                >
                  <div
                    class="undefined"
                  >
                    <button
                      class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
                      type="button"
                    >
                      <svg
                        class="false h-5 w-5 text-white -ml-1 mr-2"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M7.197,16.963H7.195c-0.204,0-0.399-0.083-0.544-0.227l-6.039-6.082c-0.3-0.302-0.297-0.788,0.003-1.087 C0.919,9.266,1.404,9.269,1.702,9.57l5.495,5.536L18.221,4.083c0.301-0.301,0.787-0.301,1.087,0c0.301,0.3,0.301,0.787,0,1.087 L7.741,16.738C7.596,16.882,7.401,16.963,7.197,16.963z"
                          fill="none"
                        />
                      </svg>
                      Login
                    </button>
                  </div>
                  <div
                    class="undefined"
                  >
                    <button
                      class="undefined inline-flex items-center px-3 py-2 text-sm rounded-lg shadow-sm   font-medium text-white bg-indigo-600 hover:bg-indigo-700 hover:shadow-lg disabled:opacity-50 disabled:cursor-not-allowed text-center"
                      type="button"
                    >
                      <svg
                        class="false h-5 w-5 text-white -ml-1 mr-2"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M6 18L18 6M6 6l12 13"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Clear
                    </button>
                  </div>
                </span>
                <h4
                  class="text-center mt-2 text-white"
                >
                   
                  <b>
                    Note
                  </b>
                  : After 3 invalid login attempts, accounts will be locked.
                </h4>
                <h4
                  class="text-center text-white"
                >
                  In that case contact the Support Team.
                </h4>
              </div>
              <div
                class="flex p-4 flex-row items-center justify-around"
              >
                <div
                  class="flex mt-2 justify-center items-center"
                >
                  <a
                    class="text-white mr-2"
                    href="#"
                  >
                    Forgot Password
                  </a>
                  <a />
                </div>
                <div>
                  <a
                    class="text-white"
                    href="privacy-policy"
                  >
                    Privacy and Policy
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="flex flex-row justify-center"
      >
        <a
          href="https://appho.st/d/VZXlvzKV"
        >
          <img
            class="h-15"
            src="app-ico.png"
          />
        </a>
            
        <a
          href="https://apps.apple.com/us/app/memetoons/id1517184743"
        >
          <img
            class="h-15"
            src="app-ico2.png"
          />
        </a>
      </div>
      <div
        class="container"
      />
      <div
        class="container"
      />
      <div
        class="container"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
